### PR TITLE
Convert ViewStyles to data class

### PR DESCRIPTION
**Pull request by JcMinarro**
### Description

Convert ViewStyles to data class
### Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- ~[ ] Changelog updated with client-facing changes~
- ~[ ] New code is covered by unit tests~
- ~[ ] Comparison screenshots added for visual changes~
- [x] Reviewers added

### CODE REVIEW

**Review by zsmb13 — COMMENTED** [review]
We have a couple `var`s in these classes, can we make those `val`s while we're here?

**Review by bychkovdmitry — APPROVED** [review]

**Review by zsmb13 — APPROVED** [review]
LGTM

**Comment by bychkovdmitry** [comment]
> We have a couple `var`s in these classes, can we make those `val`s while we're here?

Won't compile

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/MessageListViewStyle.kt [hunk 1]
```diff
@@ -73,10 +73,4 @@ public class MessageListViewStyle(
             }
         }
     }
-
-    // TODO: why this method changes the style?
```
Comment (zsmb13) [inline stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/MessageListViewStyle.kt:77]: 😃

**Pull request merged**